lib/examples/pdsch_ue.c

```diff
@@ -504,13 +504,14 @@ int main(int argc, char** argv)
       exit(0);
     }
 
+    #define MAX_SRATE_DELTA 2 // Hz allowable delta between requested and actual sample rate
```

Comment:
I'd move the define also up in to where all other macros are defined

Reply:
OK - I didn't like it here eithe... but didn't know whether it should be promoted into a `.h` file...or just placed at the head of the `pdsch_ue.c`. I'll go with the latter unless you advise otherwise